bus/bus.go

```diff
@@ -529,6 +550,8 @@ func New(s Syncer, cm ChainManager, tp TransactionPool, w Wallet, hdb HostDB, cs
 // NewServer returns an HTTP handler that serves the renterd store API.
 func NewServer(b *Bus) http.Handler {
 	return jape.Mux(map[string]jape.Handler{
+		"GET    /health": b.healthHandler,
```

Comment:
hmmmmmmm

I'm gonna invoke YAGNI on this. Easy to add later. I don't see the value in adding it now.